docker-compose.yml

```diff
@@ -1,31 +1,54 @@
 version: "3.7"
 services:
   neo4j:
-    image: neo4j:3.5
+    image: neo4j:4.4.5-community
     restart: unless-stopped
     ports:
       - 7474:7474
       - 7687:7687
+    # expose:
+    #   - 7687
     volumes:
       - ./.compose/neo4j/conf:/conf
       - ./.compose/neo4j/data:/data
       - ./.compose/neo4j/import:/import
       - ./.compose/neo4j/logs:/logs
       - ./.compose/neo4j/plugins:/plugins
     environment:
-      # Raise memory limits
+      # Raise memory limits:
       - NEO4J_dbms_memory_pagecache_size=1G
       - NEO4J_dbms.memory.heap.initial_size=1G
       - NEO4J_dbms_memory_heap_max__size=1G
+      # Auth:
+      - NEO4J_AUTH=none
+      # Add APOC and GDS:
+      - apoc.export.file.enabled=true
+      - apoc.import.file.enabled=true
+      - apoc.import.file.use_neo4j_config=true
+      - NEO4JLABS_PLUGINS=["graph-data-science", "apoc"]
+      - NEO4J_dbms_security_procedures_whitelist=gds.*, apoc.*
+      - NEO4J_dbms_security_procedures_unrestricted=gds.*, apoc.*
+      # Networking:
+      - dbms.connector.bolt.listen_address=0.0.0.0:7687
+
     healthcheck:
         test: ["CMD", "curl", "-f", "http://localhost:7474"]
-        interval: 30s
+        interval: 10s
         timeout: 10s
-        retries: 5
+        retries: 10
   cartography:
-    image: lyft/cartography
-    user: cartography
+    build:
+      context: .
+      dockerfile: dist.Dockerfile
     init: true
     restart: on-failure
     depends_on:
       - neo4j
+    environment:
+      - EXPERIMENTAL_NEO4J_4X_SUPPORT=True 
+    volumes:
+      - /Users/stevesolun/.aws:/root/.aws/
```

Comment:
I don't think this is resolved. It's still using your home directory for the volume mount :)

Reply:
I will check again and re-PR it